ocs_ci/helpers/odf_cephfs_snap.py

```
@@ -158,6 +158,39 @@ def _teardown():
+    consumer_ocp = ocp.OCP(kind=constants.STORAGECONSUMER, namespace=consumer_ns)
+    for name in get_consumer_names():
+        consumer_data = consumer_ocp.get(resource_name=name)
+        if (
+            consumer_data.get("status", {}).get("client", {}).get("name")
+            == storage_client_name
+        ):
+            return name
+    raise AssertionError(
+        f"No StorageConsumer found for storage client "
+        f"'{storage_client_name}' on provider"
+    )
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)